docs/src/main/asciidoc/sending-emails.adoc

```diff
@@ -63,7 +63,7 @@ quarkus.mailer.password=....
 
 The following table lists the properties that can be configured:
 
-include::generated/quarkus-mailer-impl.adoc[]
+include::generated/quarkus-mailer-impl.adoc[opts=optional, leveloffset=+1]
```

Comment:
Same here, I would move the configuration reference at the end and add a note pointing to it.